### PR TITLE
test at fixing url's so we use dashes everywhere!!

### PR DESCRIPTION
**Pull request by lwasser**
hey @choldgraf are you able to look at this and tell me what i'm doing wrong with renaming? my goal is to get more SEO friend url's but what i actually did was break all of the review process docs (they are no longer rendering). many thanks. i'm sure this is user error!!

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
+- file: open-source-software-peer-review/aims_scope
   sections:
-    - file: authoring/overview
-    - file: authoring/testing
-    - file: authoring/collaboration
-    - file: authoring/release
-
-- file: process/intro
-  expand_sections: true
+  - file: open-source-software-peer-review/guidelines
```
Comment (lwasser) [inline _toc.yml:6]: here i renamed the source dir so get a nice expressive path / url.

File: _toc.yml [hunk 2]
```diff
@@ -1,28 +1,22 @@
...
   sections:
   - file: submissions/author_guide
+  - file: submissions/intro
   - file: submissions/reviewer_guide
-  - file: submissions/editor_guide
-
 - file: appendices/templates
-- file: appendices/glossary
-
+  sections:
```
Comment (lwasser) [inline _toc.yml:21]: attempting to force the order. but it seems to want to order itself when i build locally.

**Comment by choldgraf** [comment]
hmm - it seems to look OK to me in the built docs: https://218-165885836-gh.circle-artifacts.com/0/html/open-source-software-peer-review/index.html

Those don't look correct to you?

**Comment by lwasser** [comment]
@choldgraf so weird. on my computer it looks like it was not in the correct order. so maybe i'll merge this now and then @xmnlab can grab these change from master. locally the appendix was not at the end in my build. circleci looks great. thank you. i'll have to get my local build straight.

**Comment by choldgraf** [comment]
@lwasser you might be running into cacheing errors - try deleting the `_build` folder and see how it looks when you re-build

**Pull request merged**